packages/discovery-provider/plugins/notifications/src/email/notifications/renderEmail.ts

```diff
@@ -86,16 +87,12 @@ const DEFAULT_TRACK_COVER_ART_URL = ''
 const DEFAULT_PLAYLIST_IMAGE_IRL = ''
 
 const getUserProfileUrl = (user: UserResource) => {
-  if (!user.creator_node_endpoint) {
-    return null
-  }
-  const contentNodes = user.creator_node_endpoint.split(',')
```

Comment:
big yikes. nice. can you check if user has `.cover_photo` and `.profile_picture` ? if they are coming from our api they should just full content-node urls

Reply:
don't think we have those it's just whatever is in the discovery DB
https://github.com/AudiusProject/audius-protocol/blob/564399399819f15046536b42a86a6492e4b20076/packages/discovery-provider/plugins/notifications/src/email/notifications/renderEmail.ts#L129-L141

Reply:
ah word, yup this good then :)